src/Components/Instructions/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export class Instructions extends React.Component {
```

Comment:
I didn't know you could directly export a class like this. Every day's a learning day!